### DISCUSSION TITLE
Heavily Yellow Tinted Screen

### DISCUSSION THREAD

**Discussion by ossiemoore** [tiny-pilot/tinypilot#468] [Q&A] [answered]
Has anyone seen this when connecting to a remote computer? While I am really pleased with TinyPilot, this one issue is an annoyance. Any ideas what's going on and how to remedy this? Happens with any remote workstation I connect it to. I tried the to no avail:

- Connected from macOS (Safari & Firefox)
- Connected from Windows (Firefox & Chrome) 
- Different HDMI cables

The screen shot shows the TinyBrowser window on the right and (for comparison) GitHub in the local browser on the left.

<img width="792" alt="Screen Shot 2021-02-04 at 1 55 05 AM" src="https://user-images.githubusercontent.com/44754148/106862106-45f3eb00-668c-11eb-84b8-e921cfacb2c9.png">

**Comment by mtlynch** [comment, marked as answer]
Thanks for reporting this!

I haven't ever seen anyone run into this before. Would it be possible to connect a physical monitor to the target computer to see if the red tint is there?

Is it possible that the [Night light setting](https://support.microsoft.com/en-us/windows/set-your-display-for-night-time-in-windows-10-18fe903a-e0a1-8326-4c68-fd23d7aaf136) is enabled on the machine that tints the screen red? A program like [f.lux](https://justgetflux.com/) might do the same thing.

**Comment by ossiemoore** [comment]
I suspected the Night light setting thing and looked at that right away. It was turned off. However, I decided to double check after you suspected that as well. It is indeed *scheduled* to kick on at 9pm. So perhaps I was looking at the wrong thing. It is fine now.

Thanks for the reply. You can kill this issue.

**Reply by mtlynch** [reply]
Thanks for letting me know!

**Reply by ossiemoore** [reply]
Note .. weirdly.. my monitor did *not* exhibit the behavior. Anyway.. very weird but on to bigger/better things. So even with it showing as "off", perhaps the OS scheduler was getting confused with the HDMI data source of the Pi looking different and maybe would switch "on" for that different display. Will play a bit with it later tonight and let you know just so that you have more information should someone have the same odd behavior.
